### DISCUSSION TITLE
Let us permanently turn off recommendations

### DISCUSSION THREAD

**Discussion by mcclure** [community/community#70468] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Logged in to Github on a new machine.

Got a new copy of the announcement box on the front page:

"We've combined the power of the Following feed with the For you feed so there’s one place to discover content on GitHub. There’s improved filtering so you can customize your feed exactly how you like it, and a shiny new visual design. ✨"

Since you felt like telling me this again, I'm going to tell you again: I don't want this. I didn't want you to combine the Following feed with the For You feed, I wanted you to remove the For You feed entirely. The For You feed should not exist. You should not have built it, you should not have shown it to me, and you should not be forcing me to use it. Every time I see this I am reminded that you do not care about me as a user, that you see me as cattle to be herded, and that you bought GitHub so that you could herd me like cattle. It makes me want to not use GitHub at all.

Also, your cutesy little ✨ is just insulting. It's one thing to shove unwanted crap in our faces for corporate strategy reasons and another entirely to pretend you're doing something fun and cool rather than just controlling us.

**Comment by jackdoe** [comment]
so mean... but as the poet says, github, can you handle the truth?

**Comment by MWstudios** [comment]
I feel like GitHub is slowly drifting towards a social media-like site rather than just a repository manager. Before the Microsoft acquisition, you could do anything on your own will and the site would keep a neutral stance towards your experience, but now GitHub constantly attempts to "suggest" and "recommend" things that _they_ deem the most fitting, as well as shoving the Copilot in your face because _they_ like it, and _they_ know what's best for you.

If I discovered GitHub today, I would get repulsed immediately and avoid it by all costs. Thank god the git command itself doesn't act this way, imagine how painful would it be if it showed ads and suggestions in your terminal.

**Comment by Apreche** [comment]
100% agreement. In the feed I want to see exactly what I want to see, in reverse chronological order, and nothing else at all. No suggestions, no recommendations, no trends, no nonsense.

**Comment by francisco-core** [comment]
Please just stop the social-media nonsense. Keep it simple with the timeline of the project we follow and nothing else.
